homeassistant/components/system_bridge/media_player.py

```diff
@@ -0,0 +1,293 @@
+    def is_volume_muted(self) -> bool | None:
+        """Boolean if volume is currently muted."""
+        bridge: Bridge = self.coordinator.data
+        if bridge.media_status is None or bridge.media_status.attributes is None:
+            return None
+        return bridge.media_status.muted
+
+    @property
+    def media_image_url(self) -> str | None:
+        """Image url of current playing media."""
+        bridge: Bridge = self.coordinator.data
+        if (
+            bridge.media_status is None
+            or bridge.media_status.attributes is None
+            or not bridge.media_status.has_cover
+        ):
+            return None
+        return bridge.media_cover_url
+
+    @property
+    def media_title(self) -> str | None:
+        """Title of current playing media."""
+        bridge: Bridge = self.coordinator.data
+        if bridge.media_status is None or bridge.media_status.attributes is None:
+            return None
+        return bridge.media_status.source.get("title")
+
+    @property
+    def media_artist(self) -> str | None:
+        """Artist of current playing media, music track only."""
+        bridge: Bridge = self.coordinator.data
+        if bridge.media_status is None or bridge.media_status.attributes is None:
+            return None
+        return bridge.media_status.source.get("artist")
+
+    @property
+    def media_album_name(self) -> str | None:
+        """Album name of current playing media, music track only."""
+        bridge: Bridge = self.coordinator.data
+        if bridge.media_status is None or bridge.media_status.attributes is None:
+            return None
+        return bridge.media_status.source.get("album")
+
+    async def async_mute_volume(self, mute: bool) -> None:
+        """Mute the volume."""
+        bridge: Bridge = self.coordinator.data
+        await bridge.async_update_media("mute", {"value": mute})
+
+    async def async_set_volume_level(self, volume: float) -> None:
+        """Set volume level, range 0..1."""
+        bridge: Bridge = self.coordinator.data
+        await bridge.async_update_media("volume", {"value": volume * 100})
+
+    async def async_media_play(self) -> None:
+        """Send play command."""
+        bridge: Bridge = self.coordinator.data
+        await bridge.async_update_media("play", None)
+
+    async def async_media_pause(self) -> None:
+        """Send pause command."""
+        bridge: Bridge = self.coordinator.data
+        await bridge.async_update_media("pause", None)
+
+    async def async_media_stop(self) -> None:
+        """Send stop command."""
+        bridge: Bridge = self.coordinator.data
+        await bridge.async_update_media("stop", None)
+
+    async def async_media_seek(self, position: float) -> None:
+        """Send seek command."""
+        bridge: Bridge = self.coordinator.data
+        await bridge.async_update_media("seek", {"value": position})
+
+    # TODO: Handle browse media
+    # async def async_browse_media(
```

Comment:
Can I get some help on how to use the media browser? My use case is to supply a temporary hosted URL to the API.

Had a browse through other implementations and couldn't find one that worked..